module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,32 @@
+#include "sort.hpp"
+
+std::deque<std::string> lengthSort(const std::forward_list<std::string>& list) {
```

Comment:
Please look into `sort()` method for lists on cppreference (especially the Parameters section) :)